static/presentations/README.md

```diff
@@ -17,5 +17,5 @@
 | 2018-09-20 | [Martin Krcek](https://linkedin.com/in/martinkrcek/)       | [Alexa, let's build a voice-Bot](2018-09-20/2018-09-20-Martin_Krcek-Alexa_lets_build_a_Voice_Bot.pdf) [[youtube](https://youtu.be/6MhBmCaG3iE)]          |
 | 2018-12-05 | [Martin Krcek](https://linkedin.com/in/martinkrcek/)       | [Assistant Roadshow Prague and Healthcare](2018-12-05/2018-12-05-Martin_Krcek-Assistant_Roadshow_Prague_and_Healthcare.pdf)                              |
 | 2018-12-05 | [Vladimir Simek](https://www.linkedin.com/in/vsimek/)      | [News from re:Invent 2018](2018-12-05/2018-12-05-Vladimir_Simek-News_from_reInvent_2018.pdf)                                                             |
-|            |                                                            |                                                                                                                                                          |
+| 2019-02-13 | [Andrej Hoos](https://www.linkedin.com/in/andrej-hoos-77515564/)+[Simon Sotak](https://www.linkedin.com/in/simon-sotak-62005911/) | [Python Serverless ML Deployment on AWS](2019-02-13-Simon_Sotak-Python_Serverless_ML_Deployment_on_AWS)|
```

Comment:
The link here is not - missing date directory + `.pdf` at the end.

It should be:

```
[Python Serverless ML Deployment on AWS](2019-02-13/2019-02-13-Simon_Sotak-Python_Serverless_ML_Deployment_on_AWS.pdf)|
```

Thank you for fixing it.